tests/pulser_simulation/test_simulation.py

```diff
@@ -1973,17 +1975,60 @@ def test_detuning_noise():
     rydberg_1 = sim_samples["Local"]["ground-rydberg"]["q1"]["det"]
     digital_0 = sim_samples["Local"]["digital"]["q0"]["det"]
     digital_1 = sim_samples["Local"]["digital"]["q1"]["det"]
-    np.all(np.isclose(rydberg_0, np.array([-0.04902824] * (2 * duration + 1))))
-    np.all(np.isclose(rydberg_1, np.array([-0.04902824] * (2 * duration + 1))))
-    np.all(
+
+    assert np.all(
+        np.isclose(rydberg_0, np.array([-0.04902824] * (2 * duration) + [0.0]))
+    )
+    assert np.all(
+        np.isclose(rydberg_1, np.array([-0.04902824] * (2 * duration) + [0.0]))
+    )
```

Comment:
nit: For these cases you can use `np.testing.assert_allclose()`, e.g.
```suggestion
    np.testing.assert_allclose(
        rydberg_0, np.array([-0.04902824] * (2 * duration) + [0.0])
    )
    np.testing.assert_allclose(
        rydberg_1, np.array([-0.04902824] * (2 * duration) + [0.0])
    )
```